subprojects/language-java/src/main/java/org/gradle/external/javadoc/JavadocOptionFileOption.java

```diff
@@ -23,4 +23,7 @@
  */
 public interface JavadocOptionFileOption<T> extends OptionLessJavadocOptionFileOption<T> {
     String getOption();
+
+    @Override
+    JavadocOptionFileOption<T> duplicate();
```

Comment:
Does it make sense to have these use a copy constructor too so we don't have to have `duplicate` in the public API?